lib/ip_ranger/ip_address.rb

```diff
@@ -34,6 +34,10 @@ def last
       to_range.last.to_i
     end
 
+    def to_s
```

Comment:
To retain compatibility with the underlying `IPAddr` object, should we maybe rename this to something else? `cidr` perhaps as that is the notation we're using?